### DISCUSSION TITLE
Add new highlights to style modified and inactive buffers in the bufferline

### DISCUSSION THREAD

**Discussion by sharpenedblade** [helix-editor/helix#10405] [Ideas]
A highlight called `ui.bufferline.inactive` could be added which behaves like `ui.bufferline.active`, but only targets inactive buffers. Also add `ui.bufferline.active.modified` and `ui.bufferline.inactive.modified` for styling modified buffers. This will let themes create "tabs" that convey more information to the user using color and font styles.

It would also be nice to make the modified indicator (which is currently `[+]`) configurable, but this might belong in another issue.

**Comment by cmattinson** [comment]
I've added a pull request to allow customizing the colors on the bufferline. I can also look at adding configuation for the modified indicator but I will consider that separate.

**Comment by chtenb** [comment]
It would also make sense to have a `ui.bufferline` highlight then. Right now it's always set to be the statusline, but this may not be what everybody wants. Look at this example, which is confusing in it's current form as to what is the active buffer.

![image](https://github.com/helix-editor/helix/assets/1030961/b71b692a-97db-4da8-9f03-da0e35cbb8ab)

The inactive buffer has a grayed-out background. However, since the background for the bufferline is the same as statusbar, it almost looks like `today.md` is active, because it stands out.
